### PR TITLE
Add LabelFractionPredictor with tests

### PR DESCRIPTION
**Pull request by genfx999**
# Citrine Python PR

## Description 
https://citrine.atlassian.net/browse/PLA-3759

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [/] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [/] I have added tests for 100% coverage
- [/] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

**Comment by latture** [comment]
@genfx999 We're renaming this predictor `IngredientFractionsPredictor` and allowing multiple labels per predictor in [mithril #529](https://github.com/CitrineInformatics/mithril/pull/529/). Do you mind if I push the update to your branch?

**Comment by latture** [comment]
Sorry for the last minute rename. I'll go ahead and push the changes. We can revert later if needed.

Overall, this PR was nearly perfect. The class and tests were great. There were only two minor things missing. We need to add the class to the map in `Predictor.get_type`, and we usually add all the exported predictors to the `__all__` list at the top of `predictors.py`. I forgot to document the latter item. I added both changes to the latest commit. Thanks for picking up this card and getting the changes out so quickly.

**Comment by genfx999** [comment]
@latture We good to merge this to formulations branch?

**Comment by latture** [comment]
We just need to add user docs. I'll write those this morning.

**Comment by latture** [comment]
@gregor-robinson @maxhutch Would one of you be able to do a quick review of this PR? I've reviewed it, and it looks good to me. However, I've pushed enough code to resolve merge conflicts with the feature branch it was based off of that a second pair of eyes would be good.

File: src/citrine/informatics/predictors.py [hunk 1]
```diff
@@ -542,3 +544,62 @@ def _post_dump(self, data: dict) -> dict:
...
+    Parameters
+    ----------
+    name: str
+        name of the configuration
+    description: str
+        description of the predictor
+    input_descriptor: FormulationDescriptor
+        descriptor that contains formulation data
+    labels: List[str]
+        labels to search for
```
Comment (maxhutch) [inline src/citrine/informatics/predictors.py:561]:
```suggestion
        labels to compute the quantity fractions of
```

**Pull request merged**